### PR TITLE
[#3422]improve(doc, build): Set code tab switching params in the Python Client doc

### PR DESCRIPTION
**Pull request by xunliu**
### What changes were proposed in this pull request?

1. Update markdown file
2. Optimized Gradle script in the Python client module.

### Why are the changes needed?

1. add missing Python tab switch params in the `how-to-use-python-client.md`
3. Optimized Gravitino Python client Gradle build script

Fix: #3422

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

CI Passed

### CODE REVIEW

File: clients/client-python/setup.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 setup(
     name="gravitino",
     description="Python lib/client for Gravitino",
-    version="0.5.0.dev12",
+    version="0.5.0.dev20",
```
Comment (xunliu) [inline clients/client-python/setup.py:18]: Because I test many times in the local, must special different version number for each test.

**Comment by xunliu** [comment]
@jerryshao Please help me review this PR, thanks.

**Comment by jerryshao** [comment]
I'm confused about the meaning of "add lose automatic of select...", what do you say meaning by, can you clearly describe what you fixed?

File: clients/client-python/setup.py [hunk 2]
```diff
@@ -24,7 +24,7 @@
     python_requires=">=3.8",
     packages=find_packages(exclude=["tests*"]),
     classifiers=[
-        'Development Status :: 3 - Alpha',
+        'Development Status :: 5 - Production/Stable',
```
Comment (jerryshao) [inline clients/client-python/setup.py:27]: I don't think our current Python client reaches to level 5, why do we update this?

Comment (xunliu) [reply clients/client-python/setup.py]: OK, I rollback these change.

**Review by jerryshao — APPROVED** [review]
LGTM.

**Pull request merged**